### ISSUE TITLE
Internal error in semgrep-core: 'NoTokenLocation min_max, empty list exn'

### ISSUE THREAD

**Issue by ievans** [semgrep/semgrep#1261]
```running 1 rules...
an internal error occured while invoking semgrep-core:
\tunknown exception: (Failure
  \"NoTokenLocation min_max, empty list exn while processing sgrep_target_0ormnmzs.py for rule 0..2.2.4, with metavar $B, close location = { \\\"line\\\": 28, \\\"col\\\": 13, \\\"offset\\\": 707 }\")
An error occurred while invoking the semgrep engine; please help us fix this by creating an issue at https://semgrep.dev
```

https://semgrep.live/pKeJ?registry=python.sqlalchemy.correctness.bad-operator-in-filter

**Comment by nbrahms** [comment]
This may possibly be a duplicate of #1259

**Comment by mschwager** [comment]
EDIT: tracking this bug in https://github.com/returntocorp/semgrep/issues/1407

**Comment by nbrahms** [comment]
Not sure about @mschwager 's mentioned problem, but the semgrep.dev link here works, so looks like this has been fixed, at least in Semgrep 0.15.
